### ISSUE TITLE
Support | D10 Defects and Operationals

### ISSUE THREAD

**Issue by productmike** [department-of-veterans-affairs/va.gov-cms#14708]
## Description / User Story
As a someone on the helpdesk team, I want to understand the extent of potential impact to the editors, so that I can narrow down and create appropriate artifacts to manage those expected requests post-D10 upgrade. 

## Acceptance Criteria
- [ ] Helpdesk has logged the extent of the D10 changes and how they'll effect the editor experience.
- [ ] Helpdesk has supporting artifacts (KBAs, response templates, ability to measure volume of D10-specific tickets, etc.) planned. 
- [ ] Helpdesk has corroborated operationally with training to ensure alignment.

## Tasks
- [ ] Drupal dev, PM and Helpdesk to meet to highlight and confirm editor impacting changes ([change log](https://www.drupal.org/project/drupal/releases/10.0.0), CKEditor5 stuff).
- [ ] Helpdesk to review tugboat instance to confirm editor impacts. (inclu tugboat link)

**Comment by EWashb** [comment]
We are just hoping to identify the KBs and training materials that will need to be updated, right? This ticket could get quite large depending on what we find, so I don't want to back us into a corner with too large of a lift within one sprint of time. 

Otherwise, excited to see this on the horizon!

**Comment by productmike** [comment]
Correct @EWashb - a gap analysis to determine what might be need from the support side of things

**Comment by BerniXiongA6** [comment]
Hi @TroyA6 @stefaniefgray -- do we have a progress update on this ticket? This is one of our goals for Sprint 92 and @EWashb needs to know if this ticket is being blocked in any way or if it'll be able to be completed this sprint. Thanks! cc: @productmike

**Comment by stefaniefgray** [comment]
@BerniXiongA6 Thanks for checking in! We are currently blocked on this ticket until the D10 walkthrough that was postponed from Weds. 9/6 takes place on Fri. 9/8.

From there, we'll know which changes are coming up that will impact editors and their user experience, and can gauge which KB articles and training materials will need to be updated. cc: @EWashb 

Notes from 9/7 discussion with @TroyA6 (in progress): https://docs.google.com/document/d/1SfeWQUwx-nCQHGFqMxS0QPl0IARqBKT78RDorsWs81Q/edit?usp=sharing

**Comment by EWashb** [comment]
Thanks so much, Stef and Berni! If there's something dev or others can provide you to do some async spelunking of changes, please don't hesitate to contact them. I don't want the meeting to block you from starting your log.

**Comment by stefaniefgray** [comment]
@EWashb Thank you! Troy and I just discussed this -- we would LOVE a Tugboat instance where we can poke around D10 from the editors' perspective, and test things out as VAMC users, Vet Center users, etc. 

This would be enormously helpful, as well as any technical documentation (such as summarized list of updates and where they'll be on the site) that the devs could provide.

**Comment by EWashb** [comment]
Yes you need one! I believe there is an instance that you can use. I know @BlakeOrgan has used a Tugboat instance, and @laflannery used one as well to test. @edmund-dunn or really anyone...can you share this info with these nice folks so they can be unblocked?

**Comment by BlakeOrgan** [comment]
@stefaniefgray @TroyA6   
  
Here is the link that I used to test D10:  
https://vacms-12252-drupal-10-opslrfeufzwsvhxx2uuv1wmvjbiq7wuq.ci.cms.va.gov/

**Comment by BerniXiongA6** [comment]
Hi @stefaniefgray @TroyA6 could you provide a sprint progress on this ticket so that we know what came out of your conversation on 9/8 and where we've landed on this work? You can comment here for @EWashb and me as we're closing out this sprint. Thank you both!

**Comment by TroyA6** [comment]
Currently unable to access Tugboat instance - should be available in the morning...

**Comment by stefaniefgray** [comment]
Hi @BerniXiongA6 -- here's the rundown!

**Here's what we've got so far:**
1. Troy will be leading the effort to revamp our training materials for D10, and I'll be providing backup support and assistance with tackling the labor (Knowledge Base article updates, training scripts, updating the training videos, etc.)
2. During Friday's D10 walkthrough, we noted all of the changes coming up, mostly regarding CKEditor and style IDs.
3. @edmund-dunn has agreed to grant us a Helpdesk Tugboat instance of our own so that Troy and I can test things, take accurate screenshots for documentation updates, etc.

**Next steps for the upcoming sprint:**
1. @TroyA6 and I will need to go through his catalogue of KB articles and determine which ones need updated text and screenshots.
2. We'll need to do the same with our current stock of training videos.
3. Once we know which particular items will need to be revised/updated, we can update this ticket accordingly.

**Comment by BerniXiongA6** [comment]
Hi @stefaniefgray @TroyA6 -- Since you were able to complete exploratory work despite this ticket being partially blocked by Tugboat, we'll consider this work partially done. Please halt on your next steps highlighted above. I'll create a new ticket for future sprint work based on our team's needs and priorities and will alert the appropriate team member at that time of next steps. Thank you! cc: @EWashb

**Comment by TroyA6** [comment]
@BerniXiongA6 @stefaniefgray @EWashb 
Completed the Initial review of 54 Training Videos / KBAs.
Upon initial review, it does not appear that changes are needed.
Where the CKE is shown, nothing is demonstrated, with the exception of 2 short demos for creating links.
The only difference is cosmetic, and that is slight -- _**THOUGHT:**_ create a document highlighting the differences and send via email blast to all editors...

**Comment by BerniXiongA6** [comment]
@TroyA6 please halt on next steps. @EWashb and I will be discussing and will get back to you. Thanks!
